perftests/Xcode/PerfTests/CorePerfTests.mm

```diff
@@ -63,7 +63,7 @@ static int32_t IntFromValue(const core::ValueType& Value) {
 
   virtual void start(BuildEngine& Engine) override {
     // Request all of the inputs.
-    for (int i = 0, e = Inputs.size(); i != e; ++i) {
+    for (size_t i = 0, e = Inputs.size(); i != e; ++i) {
```

Comment:
`auto` is maybe the best choice for places where we are inheriting the type from `.size()`.